operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -679,7 +679,11 @@ function wdtCreatePrimordialDomain() {
     > ${WDT_OUTPUT}
   ret=$?
   if [ $ret -ne 0 ]; then
-    trace SEVERE "WDT Create Domain Failed ${ret}"
+    if [ "JRF" == "$WDT_DOMAIN_TYPE" ] ; then
```

Comment:
Do we need to do the same for update code path in wdtUpdateModelDomain method?

Reply:
The JRF schema data is only created and inserted during domain creation, the update usually deal with resources and app deployment, that's why it only in creates